src/musicPlayer.java

```diff
@@ -1,23 +1,23 @@
 import components.queue.Queue;
 import components.queue.Queue1L;
 
-/** */
 public class musicPlayer {
-    /** */
 
     private Queue<String> playlist;
+    private String currentTrack;
+    private boolean isPlaying;
 
-    /** */
     public musicPlayer() {
         this.playlist = new Queue1L<>();
+        this.currentTrack = null;
```

Comment:
I might not even store this. If the front of the queue is meant to be the currently playing song, then you don't need it. That will eliminate null.